.circleci/windows/erlang_install.bat

```diff
@@ -0,0 +1,20 @@
+@echo off
+IF NOT "x%1"=="x" IF NOT "x%2"=="x" GOTO:INSTALL
+@echo Usage: %0 ^<otp_version^> ^<install_path^>
```

Comment:
although this is CI script i made it user friendly so it could be reused locally by users